shopify/product-quiz-template/web/components/QuizForm.tsx

```diff
@@ -74,7 +79,9 @@ export default ({
               requiredIndicator
               label="Name"
               autoComplete="off"
-              error={errors.quiz?.title?.message}
+              error={
+                (errors.quiz as { title?: { message: string } })?.title?.message
```

Comment:
This feels like we should be passing in typing to `useFormContext` instead of using `as` here